elasticdump.js

```diff
@@ -81,8 +96,15 @@ var elasticdump = function (input, output, options) {
       self.options.transform = [self.options.transform]
     }
     self.modifiers = self.options.transform.map(function (transform) {
-      var modificationScriptText = '(function(doc) { ' + transform + ' })'
-      return new vm.Script(modificationScriptText).runInThisContext()
+      if (transform[0] === '@') {
+        return function (doc) {
+          var parsed = url.parse(transform.slice(1))
+          return require(parsed.pathname)(doc, getParams(parsed.query))
+        }
+      } else {
+        var modificationScriptText = '(function(doc) { ' + transform + ' })'
+        return new vm.Script(modificationScriptText).runInNewContext()
```

Comment:
Sorry, just realized that this reverted back to `runInNewContext`. I'll submit a PR now.